src/Engine/Scene/CameraComponent.hpp

```diff
@@ -13,6 +13,7 @@ namespace Scene {
 class Entity;
 
 /// A Camera is an Engine Component storing a Camera object.
+///\todo attached camera do not take entity's transformation into account ...
```

Comment:
```suggestion
///\todo attached camera do not take entity's transformation into account. This prevents, for now, keyframe animation of the Camera from its entity ...
```

Reply:
@MathiasPaulin actually if you have a simple example of how one can imagine animate a camera's entity, I'll can do a quick fix.